Full:
<error>
* Message: "High-level message"
* Class: `rlang_error`
* Backtrace:
█
├─rlang::catch_cnd(a())
│ ├─base::tryCatch(...)
│ │ └─tryCatchList(expr, classes, parentenv, handlers)
│ │   └─tryCatchOne(expr, names, parentenv, handlers[[1L]])
│ │     └─doTryCatch(return(expr), name, parentenv, handler)
│ └─base::force(expr)
└─a()
  └─b()
    └─c()
<error: parent>
* Message: "Low-level message"
* Class: `rlang_error`
* Backtrace:
█
└─f()
  └─g()
    └─h()

Collapsed:
<error>
* Message: "High-level message"
* Class: `rlang_error`
* Backtrace:
█
├─[ rlang::catch_cnd(...) ] with 5 more calls
└─a()
  └─b()
    └─c()
<error: parent>
* Message: "Low-level message"
* Class: `rlang_error`
* Backtrace:
█
└─f()
  └─g()
    └─h()

Branch:
<error>
* Message: "High-level message"
* Class: `rlang_error`
* Backtrace:
 ─rlang::catch_cnd(a())
 ─a()
 ─b()
 ─c()
<error: parent>
* Message: "Low-level message"
* Class: `rlang_error`
* Backtrace:
 ─f()
 ─g()
 ─h()
Call `summary(rlang::last_error())` to see the full backtrace
